src/System Application/Test/SFTP Client/app.json

```
@@ -0,0 +1,31 @@
+{
+    "id": "d367ef25-dc5e-4585-b5ec-915b78878799",
+    "name": "SFTP Client Test",
+    "publisher": "Microsoft",
+    "brief": "Tests for the SFTP Client module",
+    "description": "Tests for the SFTP Client module.",
+    "version": "28.0.0.0",
+    "privacyStatement": "https://go.microsoft.com/fwlink/?LinkId=724009",
+    "EULA": "https://go.microsoft.com/fwlink/?linkid=2009120",
+    "help": "",
+    "url": "https://go.microsoft.com/fwlink/?LinkId=724011",
+    "logo": "",
+    "dependencies": [
+        {
+            "id": "dd0be2ea-f733-4d65-bb34-a28f4624fb14",
+            "name": "Library Assert",
+            "publisher": "Microsoft",
+            "version": "28.0.0.0"
+        }
+    ],
```

Missing direct dependency on the SFTP Client itself.
```suggestion
    "dependencies": [
        {
            "id": "dd0be2ea-f733-4d65-bb34-a28f4624fb14",
            "name": "Library Assert",
            "publisher": "Microsoft",
            "version": "28.0.0.0"
        },
        {
            "id": "016a19f8-20b7-4c7a-8259-658d67e40129",
            "name": "SFTP Client",
            "publisher": "Microsoft",
            "version": "28.0.0.0"
        }
    ],
```